docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -50,35 +50,25 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.
 
-2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
+2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the Service Instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
 
-3. Create a function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [function](./assets/lambda.yaml).
+3. Create a function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [function](./assets/function.yaml).
 
-4. Expose your function:
+4. Expose your function following [this](/components/serverless/#tutorials-expose-a-function-with-an-api-rule) tutorial. Follow the instructions in the `Console UI` tab.
 
-    a. In the left navigation panel, go to the **Functions** tab and click the `call-order-service` function.
-
-    b. In the **Settings & Code** section, click the **Select Function Trigger** button and expose your function via HTTPS.
-
-    c. Untick the **Enable authentication** field as there is no need to secure the connection for the purpose of this tutorial. Click **Add**.
-
-    d. Scroll down to the end of your function view and bind your function to your instance by clicking the **Create Service Bindings** button in the **Service Bindings** section. Choose the ServiceInstance you want to bind your function to, and click **Create Service Bindings**.
-
-    e. Save the settings in the right top-right corner of the page.
-
-    f. Click the **Functions** tab and wait until the function status is completed and marked as `RUNNING`.
+5. Bind the Service Instance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
```

Comment:
```suggestion
5. Bind the ServiceInstance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
```

I did mean that lack of space in `ServiceInstance` 😄 Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms).